src/QueryBuilder/DDLQueryBuilderInterface.php

```diff
@@ -274,8 +271,8 @@ public function dropCheck(string $table, string $name): string;
     /**
      * Builds an SQL statement for dropping a DB column.
      *
-     * @param string $table The table whose column is to dropp.
-     * @param string $column The name of the column to dropp.
+     * @param string $table The table whose column is to drop.
+     * @param string $column The name of the column to drop.
```

Comment:
```suggestion
     * @param string $column The name of the column to be dropped.
```

Reply:
Why? We're following MS style guide and it says to avoid passive voice.